agent/agent.go

```diff
@@ -126,6 +126,8 @@ func (a *Agent) Start() {
 		pipeline.AddEnhancer(enhancers.NewNeutronFlowEnhancer(a.Graph, cache))
 	}
 
+	pipeline.AddEnhancer(enhancers.NewSocketInfoEnhancer(expireTime*2, cleanup))
```

Comment:
This going to add some overhead so I suggest to enable extra flow enhancers like tcpMetric through the API

Reply:
the overhead is limited to the 1st packet only of each flow

Reply:
Yes but it is an overhead anyway and I think it is a good habbit to keep the same performance adding extra info through API

Reply:
yep I will do the patch on top the TCP metric to enable via capture API